cypress/component/BrDisplayUser.cy.tsx

```diff
@@ -0,0 +1,24 @@
+import React = require("react");
+import BrDisplayUser from "../../components/brickroom/BrDisplayUser";
+import "../../styles/globals.scss";
+
+describe("AddContributors.cy.tsx", () => {
+    it("something", () => {
+        /**
+         * Setup
+         */
+
+        // Mounting component
+        const name = "Mino";
+        const location = "Bari";
+        cy.mount(<BrDisplayUser id="1" name={name} location={location} />);
+
+        // Visibility check
+        cy.contains(name).should("be.visible");
+        cy.contains(location).should("be.visible");
+        cy.get("svg").should("be.visible");
+
+        // Click on link does not work because <link> works inside the app
```

Comment:
Then if it works in an e2e, remember to add a test even on the e2e side.
The scope of the testing is to assure that the component works ;) so let's get sure that works... the aesthetics of the code/structure does not matter